### PR TITLE
feat: Complete UI for creating flux and connecting apps

### LINKED ISSUE #34: UI for creating flux
- Build user interface for creating flux and adding apps.
- Add navigation also

### PR DESCRIPTION
**Pull request by hiteshkundal**
# Description
- Create an event section and action section for creating flux UI
- Add navigation
- Use a mobile-first design approach
- Make create flux page completely mobile responsive 

Fixes #34 

## feat

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Upload GIF/Video for frontend PR
- [x] Up-to-date with master
- [x] All the commits are squashed into the single commit.
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by hiteshkundal** [comment]
create-flux

File: client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  makeStyles,
+  InputLabel,
+  MenuItem,
+  FormControl,
+  Select,
+} from '../../Import/Import';
+
+let minWidth = 300;
+const SimpleSelect = (props) => {
+  console.log(minWidth);
```
Comment (nimish-gupta) [inline client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js]: remove console.logs

File: client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const itemList = props.itemList;
+  const menuList = itemList.map((i) => {
+    return (
+      <MenuItem key={i.value} value={i.value}>
+        {i.displayValue}
+      </MenuItem>
+    );
+  });
+  const classes = useStyles();
+  const [age, setAge] = React.useState('');
```
Comment (nimish-gupta) [inline client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js]: age should be null by default and not empty string

File: client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js (hunk 1 above)
Comment (hiteshkundal) [reply client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js]: done

File: client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js (hunk 2 above)
Comment (hiteshkundal) [reply client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js]: It was part of the library but now fixed/

**Pull request merged**